gocyclo.go

```diff
@@ -98,6 +101,14 @@ func isDir(filename string) bool {
 	return err == nil && fi.IsDir()
 }
 
+func isIgnored(path string) bool {
+	if len(*ignore) == 0 {
+		return false
+	}
+	match, err := regexp.MatchString(*ignore, path)
```

Comment:
you should really use regexp.MustCompile here or manually panic on error. This currently behaves badly on malformed regexes